capability/policy/policy.go

```diff
@@ -91,19 +90,14 @@ func Or(stmts ...Statement) Statement {
 type wildcard struct {
 	selector selector.Selector
 	pattern  string
-	glob     glob.Glob // not serialized
 }
 
 func (n wildcard) Kind() string {
 	return KindLike
 }
 
 func Like(selector selector.Selector, pattern string) (Statement, error) {
-	g, err := glob.Compile(pattern)
-	if err != nil {
-		return nil, err
-	}
-	return wildcard{selector: selector, pattern: pattern, glob: g}, nil
+	return wildcard{selector: selector, pattern: pattern}, nil
```

Comment:
imho, you should validate the pattern here

Reply:
Possibly, you could have a `Parse` function or something like that, that return a type (`type Pattern string` for example). Then you can attach the "match" function on that type.
Really nitpicking though.

Reply:
@MichaelMure is that what you mean? https://github.com/ucan-wg/go-ucan/pull/20/commits/16ba4b392d5ab1861e264ecb2eb185602a50dbae